docs/user/build-customization.rst

```diff
@@ -4,21 +4,21 @@ Build process customization
 Read the Docs has a :doc:`well-defined build process </builds>` that works for many projects.
 We also allow customization of builds in two ways:
 
-`Extend the build process`_
-    Keep using the default build process,
-    adding your own commands.
+Extend or partially override the build process
+   Keep using the default commands for MkDocs or Sphinx,
+   but extend or override the ones you need.
 
-`Override the build process`_
-    This option gives you *full control* over your build.
-    Read the Docs supports any tool that generates HTML.
+Completely override the build process
+   This option gives you *full control* over your build.
+   Read the Docs supports any tool that generates HTML.
 
-Extend the build process
-------------------------
+Extend or override the build process
```

Comment:
```suggestion
Extend or partially override the build process
```

Reply:
build.jobs allows for both, partial and complete override, the only difference with build.commands is the way it's done and the support for apt packages.

Reply:
I liked how the items above linked to the relevant sections previously.

I don't think we need to get into specifics of partially/full overriding in the section headings. In fact, I'd be clearer with both of these items/headings:

- Customize our standard build process
- Define a build process from scratch

That is, describe the effect to the end user more than technically describe what's happening in the build.